File: factorial/factorial.java
import java.Scanner;

class Factorial
{  
  public static void main(String args[])
  { 
      int i;
      long fact=1; 
      int number = new Scanner(System.in).nextInt();   
      for(i=1;i<=number;i++)
      {    
          fact=fact*i;    
      }    
      System.out.println("Factorial of "+number+" is: "+fact);    
  }  
}  

File: factorial/go/factorial.go
package main

import "fmt"

func factorial(n int) int {
	result := 1
	for n > 0 {
		result *= n
		n--
	}
	return result
}

func main() {
	var n int
	fmt.Print("Enter the number: ")
	fmt.Scan(&n)
	output := factorial(n)
	fmt.Printf("Factorial of %d is %d\n", n, output)
}
